schemas/profile_schema.md

```diff
@@ -0,0 +1,28 @@
+# Profile Card Data Schema (International)
```

Comment:
Update the title to MyData Profile Schema. There is no need to specify International in the title as it is specified again under Country.